package.json

```
@@ -11,7 +11,7 @@
     "test:all": "ember try:each"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "8.* || >= 10.*"
```

Let's do this instead:

```suggestion
    "node": "8.* || 10.* || >= 12.*"
```

Specifically, Node 11 is currently unsupported.